docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+dns:
+  image: devries/dnsmasq
+  links:
+  - mesosmaster
+  - slave1
+mesosmaster:
+  image: redjack/mesos-master:0.21.0
+  hostname: mesosmaster
+  environment:
+    - MESOS_QUORUM=1
+    - MESOS_WORK_DIR=/var/lib/mesos
+    - MESOS_LOG_DIR=/var/log
+  volumes:
+    - /var/log/mesos:/var/log/mesos
+  ports:
+    - "5050:5050"
+slave1:
+  image: redjack/mesos-slave:0.21.0
+  hostname: slave1
+  links:
+   - mesosmaster
+  environment:
+    - MESOS_MASTER=mesosmaster:5050
+    - MESOS_EXECUTOR_REGISTRATION_TIMEOUT=5mins
+    - MESOS_CONTAINERIZERS=docker,mesos
+    - MESOS_ISOLATOR=cgroups/cpu,cgroups/mem
+    - MESOS_LOG_DIR=/var/log
+    - MESOS_RESOURCES=ports(*):[9200-9200,9300-9300]
```

Comment:
Yes. I am currently changing the scheduler so it takes the first two ports from a series of ranges and uses them as the client and transport port. When that is done we can add multiple slaves in the docker-compose file.